README.md

```diff
@@ -1,166 +1,26 @@
-
-```sh
-npm install --global npm
-```
-
-Install [Yarn](https://classic.yarnpkg.com/en/):
-
-```sh
-npm install --global yarn
-```
-
-Install [Nodegit](https://github.com/nodegit/nodegit) dependencies:
-
-```sh
-apt-get install -y python2 python3 libkrb5-dev gcc openssl libssh2-1-dev libcurl4-openssl-dev g++ make
-```
-
-Install Node dependencies ([node_modules/](./node_modules/)):
-
-```sh
-yarn install
-```
-
-### Foundry
-
-First run the command below to get `foundryup`, the Foundry toolchain installer:
-
-```sh
-curl -L https://foundry.paradigm.xyz | bash
-```
-
-Then, in a new terminal session or after reloading your `PATH`, run it to get
-the latest `forge` and `cast` binaries:
-
-```sh
-foundryup
-```
-
-Advanced ways to use `foundryup`, and other documentation, can be found in the [foundryup package](./foundryup/README.md).
-Foundry is a blazing fast, portable and modular toolkit for Ethereum application development. It consists of:
-
-- **Forge**: Library for Unit / Fuzz testing written in Solidity (see [contracts/test/](./contracts/test/)).
-- **Cast**: Library for interacting with a live Ethereum JSON-RPC compatible node, or for parsing data. A swiss army knife for interacting with EVM smart contracts, sending transactions and getting chain data.
+## [Links](#links)
 
-Need help getting started with Foundry? Read the [📖 Foundry Book](https://onbjerg.github.io/foundry-book/)
+### ⚙️ [Repository Setup](./CONTRIBUTING.md#setup)
+### 👩‍💻 [Developer Guide](./CONTRIBUTING.md)
 
-### Hardhat
+### 💡 [Motivation](https://github.com/git-consensus/docs/blob/master/MOTIVATION.md)
 
-Hardhat is an Ethereum development environment for professionals. We use the [Hardhat Network](https://hardhat.org/hardhat-network/) for Integration testing which written in Typescript. It uses Ethers.js and Mocha/Chai. See [integration/](./integration/) for how it's used in Git Consensus.
-
-On [Hardhat's website](https://hardhat.org) you will find:
-
-- [Guides to get started](https://hardhat.org/getting-started/)
-- [Hardhat Network](https://hardhat.org/hardhat-network/)
-- [Plugin list](https://hardhat.org/plugins/)
-
-### Do Things
-
-Finished [Setup](#setup)?
-
-**Run some [unit tests](<(./contracts/test/)>) with Forge:**
-
-```sh
-forge test
-```
-
-**Build contract artifacts and run [integration tests](./integration/) with Hardhat:**
-
-```sh
-yarn test
-```
-
-**Deploy to Ropsten network:**
-
-Create a [.env](./.env) file matching the variables seen in [.env.example](./.env.example)
-
-Getting fully prepared may involve getting a [INFURA_API_KEY](https://infura.io/) by signing up, and getting some test ETH on your target network via a facet.
-
-Then run:
-
-```sh
-yarn deploy --network ropsten
-```
-
-**Generate contract API docs**:
-
-Ensure `../docs` path exists for [the docs repo](https://github.com/git-consensus/docs). If it *isn't*:
-
-```sh
-git clone https://github.com/git-consensus/docs.git ../docs
-```
-
-or
```

Comment:
on CONTRIBUTING L51 we just have http method. For brevity going to remove SSH method, hopefully developers can figure it out (cc @DmitriyShepelev)